docs/guides/how_to_guides/creating_and_editing_expectations/how_to_create_custom_expectations.rst

```diff
@@ -606,7 +608,7 @@ Beginning in version 0.13, we have introduced a new API focused on enabling Modu
               execution_engine: ExecutionEngine = None,
            ):
               """Validates the given data against the set minimum and maximum value thresholds for the column max"""
-              column_max = metrics.get("column.aggregate.max")
+              column_max = metrics["column.aggregate.custom.max"]
```

Comment:
Note that without fixing this error, great expectations exception handling hides useful information and presents the below message instead.

```
ge_tutorials_fork/getting_started_tutorial_final_v3_api$ python run_checkpoint_with_custom_expectation.py 
Calculating Metrics: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 9/9 [00:00<00:00, 757.95it/s]
Error running action with name update_data_docs
Traceback (most recent call last):
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/validation_operators/validation_operators.py", line 388, in _run_actions
    action_result = self.actions[action["name"]].run(
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/checkpoint/actions.py", line 58, in run
    return self._run(
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/checkpoint/actions.py", line 935, in _run
    self.data_context.build_data_docs(
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/core/usage_statistics/usage_statistics.py", line 262, in usage_statistics_wrapped_method
    result = func(*args, **kwargs)
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/data_context/data_context.py", line 2388, in build_data_docs
    index_page_resource_identifier_tuple = site_builder.build(
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/render/renderer/site_builder.py", line 293, in build
    site_section_builder.build(resource_identifiers=resource_identifiers)
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/render/renderer/site_builder.py", line 409, in build
    resource = self.source_store.get(resource_key)
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/data_context/store/store.py", line 123, in get
    return self.deserialize(key, value)
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/data_context/store/validations_store.py", line 152, in deserialize
    return self._expectationSuiteValidationResultSchema.loads(value)
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/marshmallow__shade/schema.py", line 769, in loads
    return self.load(data, many=many, partial=partial, unknown=unknown)
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/marshmallow__shade/schema.py", line 735, in load
    return self._do_load(
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/marshmallow__shade/schema.py", line 917, in _do_load
    raise exc
great_expectations.marshmallow__shade.exceptions.ValidationError: {'results': {1: {'expectation_config': ['Field may not be null.']}}}
Traceback (most recent call last):
  File "/usr/local/google/home/jdimatteo/dev/ge_tutorials_fork/getting_started_tutorial_final_v3_api/run_checkpoint_with_custom_expectation.py", line 10, in <module>
    context.run_checkpoint(checkpoint_name="my_checkpoint_with_custom_expectation")
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/data_context/data_context.py", line 3053, in run_checkpoint
    return checkpoint.run(
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/checkpoint/checkpoint.py", line 280, in run
    action_list_validation_operator.run(
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/validation_operators/validation_operators.py", line 332, in run
    batch_actions_results = self._run_actions(
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/validation_operators/validation_operators.py", line 407, in _run_actions
    raise e
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/validation_operators/validation_operators.py", line 388, in _run_actions
    action_result = self.actions[action["name"]].run(
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/checkpoint/actions.py", line 58, in run
    return self._run(
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/checkpoint/actions.py", line 935, in _run
    self.data_context.build_data_docs(
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/core/usage_statistics/usage_statistics.py", line 262, in usage_statistics_wrapped_method
    result = func(*args, **kwargs)
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/data_context/data_context.py", line 2388, in build_data_docs
    index_page_resource_identifier_tuple = site_builder.build(
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/render/renderer/site_builder.py", line 293, in build
    site_section_builder.build(resource_identifiers=resource_identifiers)
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/render/renderer/site_builder.py", line 409, in build
    resource = self.source_store.get(resource_key)
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/data_context/store/store.py", line 123, in get
    return self.deserialize(key, value)
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/data_context/store/validations_store.py", line 152, in deserialize
    return self._expectationSuiteValidationResultSchema.loads(value)
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/marshmallow__shade/schema.py", line 769, in loads
    return self.load(data, many=many, partial=partial, unknown=unknown)
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/marshmallow__shade/schema.py", line 735, in load
    return self._do_load(
  File "/usr/local/google/home/jdimatteo/Desktop/great_expectations_custom/venv/lib/python3.9/site-packages/great_expectations/marshmallow__shade/schema.py", line 917, in _do_load
    raise exc
great_expectations.marshmallow__shade.exceptions.ValidationError: {'results': {1: {'expectation_config': ['Field may not be null.']}}}
```